dll/ntdll/ldr/ldrinit.c

```diff
@@ -2192,6 +2193,9 @@ LdrpInitializeProcess(IN PCONTEXT Context,
     /* ReactOS specific */
     LdrpInitializeProcessCompat(&OldShimData);
 
+    /* Apply export versioning hacks */
+    RosApplyAppcompatExportHacks(NtLdrEntry);
```

Comment:
You might need to reposition this after https://github.com/reactos/reactos/pull/1871/commits/029b8f2cf9912b148f3e60be0472652f26a1adb9